### DISCUSSION TITLE
Option to have more width for issue summary in interactive mode

### DISCUSSION THREAD

**Discussion by wongfei2009** [ankitpokhrel/jira-cli#412] [Ideas]
Currently, issue summary will be truncated to maximum width, although there is still a lot freespace in the rendering screen. It makes the interactive mode becomes less useful. Because issue summary is usually longer than the hard-coded maximum width.......

**Comment by ankitpokhrel** [comment]
Hi @wongfei2009, It is bit tricky to make just one field flexible as this would require some refactoring of current implementation. However, I've increased the max width in the [latest release](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.1.0). It's not what you requested for but...
![its_something](https://user-images.githubusercontent.com/2364546/184530301-176882bc-2685-4ece-b8a1-41ddde6c0a20.jpg)

**Comment by wongfei2009** [comment]
Not bad, thank you~
